data/wiki/hello-world.md

```diff
@@ -142,9 +108,10 @@ Congratulations! You've just built and run your very first unikernel!
 
 ### Step 1: Hello World!
 
-Most programs will depend on some system devices which they use to interact with
-the environment. In this section, we illustrate how to define unikernels that
-depend on such devices. 
+Except for the noop example above, all unikernels have at least one job.
```

Comment:
This will be the best spot to define what a `job` actually is.
A simple enough definition especially for those who don't really have a background in computer sciences.....something like:
`a job typically refers to a task for our unikernel to execute`